File: demo/app/app/src/main/java/com/vectordemo/domain/model/oss/OssUploadItemModel.kt
package com.vectordemo.domain.model.oss

import com.vectordemo.utils.json.GsonBean

data class OssUploadItemModel(
    val originFileName: String,
    val success: Boolean,
    val duplicated: Boolean,
    val fileId: String,
    val url: String,
    val message: String
) : GsonBean

File: demo/flutter/flutternew/lib/domain/model/oss/oss_upload_item_model.dart
class OssUploadItemModel {
  final String originFileName;
  final bool success;
  final bool duplicated;
  final String fileId;
  final String url;
  final String message;

  const OssUploadItemModel({
    required this.originFileName,
    required this.success,
    required this.duplicated,
    required this.fileId,
    required this.url,
    required this.message,
  });
}
